suites/reef/rgw/tier-2_rgw_test_using_s3cmd.yaml

```diff
@@ -200,6 +200,16 @@ tests:
         config-file-name: ../../s3cmd/configs/test_ratelimit_tenanted_user.yaml
         timeout: 300
 
+  - test:
```

Comment:
FYI: https://bugzilla.redhat.com/show_bug.cgi?id=2262919
this BZ is moved back to assigned state
so IMO its better to add BZ[2262919](https://bugzilla.redhat.com/show_bug.cgi?id=2262919) as knownissue

Reply:
Done , added a known issue , thanks for noticing this.